docs/src/man/getting_started.md

```diff
@@ -159,14 +159,18 @@ using CSV
 using DataTables, CSV
 ```
 
-Datasets can now be read using
+Datasets can now be read from input sources using
 ```julia
-CSV.read(...)
+CSV.read(input, DataTable, args...)
```

Comment:
`args` isn't defined. I would just put the two first arguments, and say in the text that more optional arguments can be passed.